freqtrade/tests/__init__.py

```diff
@@ -0,0 +1,21 @@
+import json
+import os
+
+
+def load_backtesting_data(ticker_interval: int = 5):
+    path = os.path.abspath(os.path.dirname(__file__))
+    result = {}
+    pairs = [
+        'BTC_BCC', 'BTC_ETH', 'BTC_MER', 'BTC_POWR', 'BTC_ETC',
+        'BTC_OK', 'BTC_NEO', 'BTC_EMC2', 'BTC_DASH', 'BTC_LSK',
+        'BTC_LTC', 'BTC_XZC', 'BTC_OMG', 'BTC_STRAT', 'BTC_XRP',
+        'BTC_QTUM', 'BTC_WAVES', 'BTC_VTC', 'BTC_XLM', 'BTC_MCO'
+    ]
```

Comment:
I hope 20 coins is not slowing hyperopting too badly.

Reply:
I reduced the number of coins to 10